alpha/scripts/compatCheck.php

```diff
@@ -624,14 +624,19 @@ function compareResults($resultNew, $resultOld)
 
 	if (!$resultNew)
 	{
-		return array('failed to parse new XML');
+		return array('#ERROR - failed to parse new XML');
```

Comment:
The alert is really needed only when the old is parse and the new fails.
Otherwise there are many api's that both now and old failed on parse

Reply:
look a few lines above... this what happens.